tools/ci_build/github/azure-pipelines/templates/win-web-ci.yml

```diff
@@ -99,10 +99,6 @@ jobs:
      npm ci
     workingDirectory: '$(Build.SourcesDirectory)\js'
     displayName: 'npm ci /js/'
-  - script: |
-     npm run lint
-    workingDirectory: '$(Build.SourcesDirectory)\js'
-    displayName: 'run ESLint without TS type populated'
   - script: |
      npm ci
     workingDirectory: '$(Build.SourcesDirectory)\js\common'
```

Comment:
You may consider consolidating all these scripts into a single ps1 file so that we can run it locally.

Reply:
a .ps1 file is probably not easy to run at linux machine without installing dependencies. maybe think about using a nodejs script instead in future PR.